packages/ckeditor5-font/tests/ui/colorui.js

```diff
@@ -181,7 +181,8 @@ describe( 'ColorUI', () => {
 		describe( 'color picker', () => {
 			it( 'should execute command if the color gets changed when dropdown is open', () => {
 				const spy = sinon.spy( editor, 'execute' );
-				dropdown.colorTableView.colorPickerPageView.colorPickerView.color = '#a37474';
+
+				dropdown.colorTableView.colorPickerPageView.colorPickerView._debounceColorPickerEvent( '#a37474' );
```

Comment:
Same here: let's fire a proper event here. All we want to know is that there's a proper response to the public event being fired. 

At this stage, we don't care about the internal state of the `ColorPickerView`. I know that `_debounceColorPickerEvent()` sets both `#color` and fires the event but the latter does not matter here.